tests/qos/qos_sai_base.py

```diff
@@ -2035,6 +2035,16 @@ def skip_check_for_hbm_either_asic(self, get_src_dst_asic_and_duts):
         yield
         return
 
+    @pytest.fixture(scope="function", autouse=False)
+    def skip_src_dst_different_asic(self, get_src_dst_asic_and_duts):
+        src_asic = get_src_dst_asic_and_duts['src_asic']
+        dst_asic = get_src_dst_asic_and_duts['dst_asic']
+        if src_asic.sonichost.facts['platform'] != dst_asic.sonichost.facts['platform']:
```

Comment:
@zhixzhu does not looks correct. Need to check for asic type and not platform,

Reply:
Updated. skip_src_dst_different_asic checks asic type.